docker-compose-reload.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+services:
+
+  appwrite:
+    container_name: appwrite
+
+  live-reloader:
+    image: apogiatzis/livereloading
+    container_name: livereloader
+    privileged: true
+      environment:
+        - RELOAD_CONTAINER=appwrite
```

Comment:
I think the the delay here should be much lower, waiting 20 seconds for container to reload after a change can be forever. Maybe 1 second to start with?